src/js/UserManager.js

```diff
@@ -0,0 +1,18 @@
+import { createUserManager } from 'redux-oidc'
+
+const userManager = createUserManager({
+  authority: 'https://192.168.99.100:8443/auth/realms/hmda',
```

Comment:
I'll hold this for now, I think it would make a lot of sense to do when we switch to `hmda-platform`'s docker-compose